backend/src/API/candidateDeciderAPI.ts

```diff
@@ -4,6 +4,11 @@ import PermissionsManager from '../utils/permissionsManager';
 
 const candidateDeciderDao = new CandidateDeciderDao();
 
+/**
+ * Retrieves all Candidate Decider instances, filtering based on the user's role and permissions.
```

Comment:
Only retrieves the metadata and not the data itself. Also doesn't hurt to mention in more detail how the permission work.